tests/components/sunweg/conftest.py

```diff
@@ -86,5 +153,57 @@ def plant_fixture_alternative(inverter_fixture) -> Plant:
         0.012296,
         None,
     )
-    plant.inverters.append(inverter_fixture)
+    plant.inverters.append(inverter_fixture_alternative)
     return plant
+
+
+@pytest.fixture
+def plant_fixture_total_power_0() -> Plant:
+    """Define Plant fixture."""
+    return Plant(
+        123456,
+        "Plant #123",
+        29.5,
+        0.5,
+        0,
+        12.786912,
+        24.0,
+        "kWh",
+        0.0,
+        0.012296,
+        None,
+    )
+
+
+@pytest.fixture
+def plant_fixture_total_power_none() -> Plant:
+    """Define Plant fixture."""
+    return Plant(
+        123456,
+        "Plant #123",
+        29.5,
+        0.5,
+        0,
+        12.786912,
+        24.0,
+        "kWh",
+        None,
+        0.012296,
+        None,
+    )
+
+
+@pytest.fixture
+def api_fixture(plant_fixture):
+    """Mock APIHelper."""
+    with patch.multiple(
+        "sunweg.api.APIHelper",
```

Comment:
Please patch at the place where we use this, so `homeassistant.components.sunweg.*`